apps/openchallenges/data-lambda/pyproject.toml

```diff
@@ -12,6 +12,7 @@ dependencies = [
     "gspread==6.1.4",
     "pandas==2.2.3",
     "numpy==2.1.0",
+    "pymysql>=1.1.1",
```

Comment:
I recommend pinning the version number as done for the other packages. https://docs.renovatebot.com/dependency-pinning/

Reply:
That makes a lot of sense - will do! Thanks.